openff/evaluator/tests/test_workflow/test_workflow.py

```diff
@@ -313,3 +316,13 @@ def test_find_relevant_gradient_keys(tmpdir):
 
     assert len(gradient_keys) == len(expected_gradient_keys)
     assert {*gradient_keys} == expected_gradient_keys
+
+
+def test_generate_default_metadata_defaults():
+    dummy_property = create_dummy_property(Density)
+    dummy_forcefield = "smirnoff99Frosst-1.1.0.offxml"
+    data = Workflow().generate_default_metadata(dummy_property,
```

Comment:
```suggestion
    data = Workflow.generate_default_metadata(dummy_property,
```

this should fix the non OE tests I think!